docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,239 @@
+
+.. note::
+
+    A quasi-probability distribution differs from a probability distribution in that negative values are also allowed.
+    However the quasi-probabilities must sum up to 1 like probabilities.
+    Negative quasi-probabilities may appear when using error mitigation techniques.
+
+.. testcode::
+
+    quasi_dist = result.quasi_dists[0]
+    print(quasi_dist)
+
+.. testoutput::
+
+    {0: 0.4999999999999999, 3: 0.4999999999999999}
+
+Probability distribution with binary outputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you prefer to see the outputs as binary strings instead of decimal, you can use the
+:meth:`~qiskit.result.QuasiDistribution.binary_probabilities` method.
+
+.. testcode::
+    
+    print(quasi_dist.binary_probabilities())
+
+.. testoutput::
+
+    {'00': 0.4999999999999999, '11': 0.4999999999999999}
+
+Parameterized circuits with ``Sampler``
+=========================================
+
+The :class:`~qiskit.primitives.Sampler` primitive also has the option to include unbound parameterized circuits like the one below.
+You can also bind values to the parameters of the circuit and follow the steps
+of the previous example.
+
+.. testcode::
+
+    from qiskit.circuit import Parameter
+
+    theta = Parameter('θ')
+    param_qc = QuantumCircuit(2)
+    param_qc.ry(theta, 0)
+    param_qc.cx(0,1)
+    param_qc.measure_all()
+    print(param_qc.draw())
+
+.. testoutput::
+
+            ┌───────┐      ░ ┌─┐   
+       q_0: ┤ Ry(θ) ├──■───░─┤M├───
+            └───────┘┌─┴─┐ ░ └╥┘┌─┐
+       q_1: ─────────┤ X ├─░──╫─┤M├
+                     └───┘ ░  ║ └╥┘
+    meas: 2/══════════════════╩══╩═
+                              0  1 
+
+The main difference from the previous case is that now you need to include the parameter values
+for which you want to evaluate the expectation value as a ``list`` of ``list``\ s of ``float``\ s.
+The idea is that the ``i``-th element of the bigger ``list`` is the set of parameter values
+that corresponds to the ``i``-th circuit.
+
+.. testcode::
+
+    import numpy as np
+
+    parameter_values = [[0], [np.pi/6], [np.pi/2]]
+
+    job = sampler.run([param_qc]*3, parameter_values=parameter_values)
+    dists = job.result().quasi_dists
+
+    for i in range(3):
+        print(f"Parameter: {parameter_values[i][0]:.5f}\t Probabilities: {dists[i]}")
+
+.. testoutput::
+
+    Parameter: 0.00000	 Probabilities: {0: 1.0}
+    Parameter: 0.52360	 Probabilities: {0: 0.9330127018922194, 3: 0.0669872981077807}
+    Parameter: 1.57080	 Probabilities: {0: 0.5000000000000001, 3: 0.4999999999999999}
+
+Change run options
+==================
+
+It is also possible that you may want to change any other option.
+
+For example, in the previous sections the :class:`~qiskit.primitives.Sampler`
+is :class:`~qiskit.quantum_info.Statevector`-based but it can be changed
+to shot-based by setting a number of ``shots``. For reproducibility purposes, in this
+guide a ``seed`` will also be set.
```

Comment:
```suggestion
By default, the reference :class:`~qiskit.primitives.Sampler` class performs an exact statevector
calculation based on the :class:`~qiskit.quantum_info.Statevector` class. However, this can be 
modified to include shot noise if the number of ``shots`` is set. 
For reproducibility purposes, a ``seed`` will also be set in the following examples.
```